app/src/main/java/com/github/geohunt/app/ui/components/statistics/ClaimPointsGraph.kt

```diff
@@ -0,0 +1,76 @@
+package com.github.geohunt.app.ui.components.statistics
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.github.geohunt.app.model.database.api.Claim
+import java.time.LocalDate
+
+/**
+ * Represents a Graph that show the evolution of the points of claims
+ * Mostly calls the createEntries method which converts a list of claims to a pair
+ * of lists (dates, points) that will represent the points in the graph we want to draw
+ * @param claims The full list of claims we want to represent of the graph
+ * @param dateGranularity the amount of claims we want to show on the graph (last week/month/year)
+ */
+@Composable
+fun ClaimPointsGraph(claims: List<Claim>, dateGranularity: DateGranularity) {
```

Comment:
It would be a good idea for this function should check if claims list is not null.

You can add at the beginning of the function the code like:
`    check(claims.isNotEmpty()) { "Claims list must not be empty" }
`